ui/store/action-queue/index.test.js

```diff
@@ -0,0 +1,197 @@
+import sinon from 'sinon';
+
+import {
+  __TEST_CLEAR_QUEUE,
```

Comment:
This is not beautiful, but currently absolutely necessary.

A change to how we invoke tests could avoid this, but the next person to add a new test would stumble upon this issue of having a prepopulated queue again.

Another possibility - expose a factory function from queue and create a new queue for each test and only one queue in actual implementation.

I just got this idea. Will implement later. Unless someone beats me to it.

Reply:
This exposes a private function in public API to facilitate testing. 

The suggestion is just to make it possible to have more instances of action-queue but with a clean API. 

If you want to avoid changing implementation to facilitate tests, you want the latter (it;s a matter of wrapping almost the whole file in an init function that reveals the api)